installinator/src/reporter.rs

```diff
@@ -137,15 +121,15 @@ where
             })
             .cloned()
             .collect();
-        // IMPORTANT: You might be tempted to replace this reverse call with
+        // STEP: You might be tempted to replace this reverse call with
```

Comment:
I think this change might've been an accident? :)

Reply:
Yes. I got rid of this code and comment with the more principled approach in #2890.